servers/cromwell/README.md

```diff
@@ -3,23 +3,37 @@
 Thin shim around [`cromwell`](https://github.com/broadinstitute/cromwell).
 
 ## Development
-For cromwell setup see the Cromwell README:
+
 1. Set the `CROMWELL_URL` environment variable to specify which cromwell instance to use. Job Manager pulls workflow data from `https://cromwell.mint-dev.broadinstitute.org/api/workflows/v1` by default:
-```
-export CROMWELL_URL=https://cromwell.test-cromwell.broadinstitute.org/api/workflows/v1
-```
 
-**Note:** If you want to setup this API Server against a locally hosted Cromwell instance, you need to explicitly provide the ip address (inet if the Cromwell is hosted on the same machine) to the Cromwell with port numbers, for example:
-```
-export CROMWELL_URL=http://192.168.0.106:8000/api/workflows/v1
-```
+    ```
+    export CROMWELL_URL=https://cromwell.test-cromwell.broadinstitute.org/api/workflows/v1
```

Comment:
Sorry - which one should I have access to? @dshiga gave us access to the "Verily" instance, which I appear to be able to access via the browser. However, using the config.json my server just fails:

```
HTTPError: 405 Client Error: Method Not Allowed for url: https://cromwell-verily.dsp-techops.broadinstitute.org/api/v1/workflows/query
```

Reply:
As discussed over slack, the issue here was that the CROMELL_URL needs to be `https://cromwell-verily.dsp-techops.broadinstitute.org/api/workflows/v1`